### PR TITLE
removes all links to broken sites

### PR DESCRIPTION
**Pull request by luketlancaster**
Address [issue #31](https://github.com/NCUSGL/ncusgl.github.com/issues/31), removing all broken links.

Left the IRC channels, in case folks wanted to hit those up.

### CODE REVIEW

**Comment by jwoodcock** [comment]
Looks like you got the links, but left the IRC channels. Is that intended?

**Comment by luketlancaster** [comment]
Figured the channels would still be up there. I'll take 'em down if that's what's wanted, but thought they'd be pretty benign. Just say the word!

File: index.html [hunk 1]
```diff
@@ -126,20 +119,16 @@
             <td width="50%" valign="top">
                 <p>Conference/Initiatives</p>
 
-                <p><a href="http://co-op.nashvl.org">The Code Cooperative</a></p>
-
                 <p><a href="http://coderfaire.com">CoderFaire</a>
```
Comment (wogsland) [inline index.html:122]: This one appears to redirect to an ad...

File: index.html [hunk 2]
```diff
@@ -126,20 +119,16 @@
...
                 <br /><strong>IRC (Freenode.net) #coderfaire</strong></p>
 
-                <p><a href="http://hacknashville.com">HackNashville</a>
                 <br /><strong>IRC (Freenode.net) #hacknashville</strong></p>
-		
-		<p><a href="https://phreaknic.info">PhreakNIC</a>
```
Comment (wogsland) [inline index.html:137]: This one just shouldn't be https

**Pull request closed without merging**